doc/source/timeseries.rst

```diff
@@ -968,6 +964,7 @@ particular day of the week:
 
 .. ipython:: python
 
+   d = datetime(2008, 8, 18, 9, 0)
```

Comment:
I personally prefer to `import datetime`, and then use `datetime.datetime(...)`. I think it's more explicit, and better to import modules directly.